deploy

```diff
@@ -1,20 +1,36 @@
-#!/bin/sh
+#!/bin/bash
+
+[ -z "$DEBUG" ] || set -x
 
 set -e
 
-remote_host=$1
-mode=$2
-version=$3
+REMOTE_HOST="$USER@www.rabbitmq.com"
```

Comment:
There is no need to prepend `$USER@`. ssh(1) will do that itself and you could also set the user in your ssh_config(5).